packages/camltc/camltc.0.9.5/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "1.2"
+maintainer: "romain.slootmaekers@openvstorage.com"
+author: ["Jan Doms" "Joost Damad" "Romain Slootmaekers" "Nicolas Trangez"]
+homepage: "http://github.com/toolslive/camltc"
+dev-repo: "https://github.com/toolslive/camltc.git"
+bug-reports: "http://github.com/toolslive/camltc/issues"
+build: [
+  [ "git" "submodule" "init"]
+  [ "git" "submodule" "update"]
+  ["%{make}%" "-C" "src"]
+]
+
+install:[
+  ["%{make}%" "-C" "src" "install"]
+]
+
+remove: [
+  ["ocamlfind" "remove" "camltc"]
+]
+
+depends: [
+  "ocamlfind" {build}
+  "ounit"
+  "lwt" {>= "4.0.0"}
```

Comment:
@toolslive, I glanced at the update of camltc. It looks to me like the constraint should be `>= 3.2.0`: the only new change that requires a lower bound is using `Lwt_preemptive` without `lwt.preemptive`, and that is possible since 3.2.0.